redis/src/main/scala/zio/redis/api/Keys.scala

```diff
@@ -13,72 +13,293 @@ import zio.{ Chunk, ZIO }
-
-  final def ttl(a: String): ZIO[RedisExecutor, RedisError, Duration] = Ttl.run(a)
-
-  final def typeOf(a: String): ZIO[RedisExecutor, RedisError, RedisType] = TypeOf.run(a)
-
-  final def unlink(a: String, as: String*): ZIO[RedisExecutor, RedisError, Long] = Unlink.run((a, as.toList))
-
-  final def wait_(a: Long, b: Long): ZIO[RedisExecutor, RedisError, Long] = Wait.run((a, b))
+    cursor: Long,
+    pattern: Option[Regex] = None,
+    count: Option[Long] = None,
+    `type`: Option[String] = None
+  ): ZIO[RedisExecutor, RedisError, (Long, Chunk[String])] = Scan.run((cursor, pattern, count, `type`))
+
+  /**
+   * Alters the last access time of a key(s). A key is ignored if it does not exist.
+   *
+   * @param key one required key
+   * @param keys maybe rest of the keys
+   * @return The number of keys that were touched.
+   * @see <a href="https://redis.io/commands/touch">https://redis.io/commands/touch</a>
+   */
+  final def touch(key: String, keys: String*): ZIO[RedisExecutor, RedisError, Long] = Touch.run((key, keys.toList))
+
+  /**
+   * Returns the remaining time to live of a key that has a timeout.
+   *
+   * @param key key
+   * @return remaining time to live of a key that has a timeout, error otherwise
+   *
+   * @see <a href="https://redis.io/commands/ttl">https://redis.io/commands/ttl</a>
+   */
+  final def ttl(key: String): ZIO[RedisExecutor, RedisError, Duration] = Ttl.run(key)
+
+  /**
+   * Returns the string representation of the type of the value stored at key.
+   *
+   * @param key key
+   * @return type of the value stored at key
+   *
+   * @see <a href="https://redis.io/commands/type">https://redis.io/commands/type</a>
+   */
+  final def typeOf(key: String): ZIO[RedisExecutor, RedisError, RedisType] = TypeOf.run(key)
+
+  /**
+   * Removes the specified keys. A key is ignored if it does not exist. The command performs the actual memory reclaiming
+   * in a different thread, so it is not blocking.
+   *
+   * @param key one required key
+   * @param keys maybe rest of the keys
+   * @return The number of keys that were unlinked.
+   *
+   * @see [[del]]
+   * @see <a href="https://redis.io/commands/unlink">https://redis.io/commands/unlink</a>
+   */
+  final def unlink(key: String, keys: String*): ZIO[RedisExecutor, RedisError, Long] = Unlink.run((key, keys.toList))
+
+  /**
+   * This command blocks the current client until all the previous write commands are successfully transferred and acknowledged
+   * by at least the specified number of replicas.
+   *
+   * @param replicas minimum replicas to reach
+   * @param timeout 0 means to block forever.
+   * @return the number of replicas reached both in case of failure and success
+   *
+   * @see <a href="https://redis.io/commands/wait">https://redis.io/commands/wait</a>
+   */
+  final def wait_(replicas: Long, timeout: Long): ZIO[RedisExecutor, RedisError, Long] = Wait.run((replicas, timeout))
```

Comment:
possibly rename to \`wait\`

Reply:
It collides with existing wait.

Reply:
Sorry, I meant with the quotes on

Reply:
I wouldn't do that, at least for now. Quoted functions are a bit awkward IMHO.

Reply:
sure 😄